tests/runtime/probe

```diff
@@ -141,6 +141,26 @@ EXPECT ERROR: Error attaching probe: kprobe:nonsense:vfs_read: specified module
 TIMEOUT 5
 WILL_FAIL
 
+NAME kprobe_wildcard_all_leading_underscore
+PROG kprobe:_* { } interval:s:1{ printf("SUCCESS"); exit(); }
+EXPECT_REGEX SUCCESS
+ENV BPFTRACE_MAX_PROBES=20000
+TIMEOUT 5
+
+NAME kprobe_disallow_rcu_functions
+PROG kprobe:rcu_* { printf("SUCCESS %d\n", pid); exit(); }
+EXPECT No probes to attach
+TIMEOUT 5
+WILL_FAIL
+
+# WILL_FAIL is required here because 'grep' returns an error code of 1 when
+# no lines are matched.
+NAME kprobe_hide_ftrace_invalid_address
+RUN {{BPFTRACE}} -l | grep -c __ftrace_invalid_address__
```

Comment:
Nit: it seems a bit more clear to me to use
```suggestion
RUN {{BPFTRACE}} -l | grep -c __ftrace_invalid_address__ || true
```
rather than `WILL_FAIL` but that may be just my preference.

Reply:
> Nit: it seems a bit more clear to me to use
> 
> rather than `WILL_FAIL` but that may be just my preference.

Removed the WILL_FAIL and took your suggestion - thanks.